setup.ts

```diff
@@ -499,6 +532,24 @@ async function main(): Promise<void> {
     await twoFactorAuth();
   }
 
+  if (process.env.LAST_RESORT_SUPERADMIN_EMAIL) {
+    console.log(
+      `\nSuper User email already exists with the value ${process.env.LAST_RESORT_SUPERADMIN_EMAIL}`
+    );
+  }
+
+  const { shouldSetSuperUserEmail } = await inquirer.prompt([
+    {
+      type: "confirm",
+      name: "shouldSetSuperUserEmail",
+      message: "Would you like to set up a super user email",
```

Comment:
Change to:

- `would you like to setup a Super Admin email of last resort?`

Reply:
commited the change.